server/src/routes.ts

```diff
@@ -535,6 +565,9 @@ export default async function routes(fastify: FastifyInstance) {
         return new createError.InternalServerError('Qr Claim does not have any event');
       }
       qr_claim.event = event;
+      if (event.event_template_id) {
+        qr_claim.event_template = await getEventTemplate(event.event_template_id);
+      }
```

Comment:
Did you try this endpoint for QRs in different states and with/out template?

Reply:
it works, but if the qr_claim event doesn't have any template you dont get the key 'event_template' in the response, in the next PR i'm adding a else to send the key in null.